docs/benchmarks.md

```diff
@@ -1,131 +1,184 @@
-| 1GB       | 1M        | 1024  | 1815                   | 9875.59       | 50.426              | 2.05                         |
-
-### Random Write
-In case of random writes, only offset will change in calls issued by fio. GCSFuse behaviour will
-remain the same and there are no changes in the way gcs calls are being made. Hence the bandwidth will be same
+
+GCSFuse performs well for sequential reads and recommendation is to use GCSFuse
+for doing sequential reads on file sizes > 10MB and < 1GB. Always use http1 (
+`--client-protocol=http1`, enabled by default) gives better throughput.
+
+## Writes
+
+### FIO spec
+  ```
+    [global]
+    ioengine=sync
+    direct=1
+    fadvise_hint=0
+    verify=0
+    # Change this to randwrite to test random writes.
+    rw=write   
+    # Update the block size value from the table for different 
+    bs=1M
+    iodepth=64
+    invalidate=1
+    time_based=0
+    file_append=0
+    # By default fio creates all files first and then starts writing to them. This option is to disable that behavior. 
+    create_on_open=1 
+    # Every file is written only once. Set nrfiles per thread in such a way that the test runs for 1-2 min. This will vary based on file size. Change the value from table to get provided results.
+    nrfiles=2
+    thread=1
+    openfiles=1
+    group_reporting=1
+    allrandrepeat=1
+    filename_format=$jobname.$jobnum.$filenum
+    [1_thread]
+    stonewall
+    # Change the test directory (1mb) for different experiments. The directory must exist within the mounted directory.
+    directory=gcs/1gb
+    # Update the file size value from table(file size) for different experiments.
+    filesize=1G  
+    numjobs=112
+ ```
+
+**Note:** Performance will be better if you experiment on scenarios where files do not exist,
+compared to scenarios where files already exist. This is because writing to an existing
+file requires reading the existing content first, before the new data can be written.
+
+### Results
+
+#### Sequential Write
+
+| File Size | BlockSize | nrfiles | Bandwidth in MiB/sec | IOPS(avg) | Avg Latency (msec) | Network Send Traffic (GiB/s) |
+|-----------|-----------|---------|----------------------|-----------|--------------------|------------------------------|
+| 256KB     | 16K       | 30      | 212                  | 14976.95  | 3.206              | 0.027                        |
+| 1MB       | 1M        | 30      | 772                  | 794.32    | 1.150              | 0.036                        |
+| 50MB      | 1M        | 20      | 3611                 | 5948.63   | 8.929              | 1.33                         |
+| 100MB     | 1M        | 10      | 3577                 | 4672.64   | 1.911              | 1.41                         |
+| 1GB       | 1M        | 2       | 1766                 | 2121.66   | 49.114             | 1.77                         |
+
+#### Random Write
+
+In case of random writes, only offset will change in calls issued by fio.
```

Comment:
Have we re-tested this statement that under the new fio spec, this statement is true?